qiskit_ibm_runtime/fake_provider/__init__.py

```diff
@@ -196,6 +197,7 @@
     FakeKolkata
     FakeLagos
     FakeLima
+    FakeAltLima
```

Comment:
We've [deprecated](https://github.com/Qiskit/qiskit-ibm-runtime/issues/1604) V1 fake backends, so there is no need for V1.